src/rethinkdb/query.cljc

```diff
@@ -223,9 +223,10 @@
   [table ids & [optargs]]
   (term :GET_ALL
         (cond
-         (sequential? ids) (concat [table] ids)
-         (map? ids)        [table (wrap-args ids)]
-         :else (throw (IllegalArgumentException. "get-all expects a collection or a map.")))
+          (sequential? ids) (concat [table] ids)
+          (map? ids)        [table (wrap-args ids)]
+          :else #?(:clj (throw (IllegalArgumentException. "get-all expects a collection or a map."))
+                   :cljs (throw (js/Error. "get-all expects a collection or a map."))))
```

Comment:
Would something like this be better? 

```
          :else (throw (#?@(:clj IllegalArgumentException. :cljs js/Error.) "get-all expects a collection or a map."))
```

(or `new #?@(...)` if the `Foo.` syntax doesn't work)

Basically, to avoid duplicating the error message. Dunno if its worth it though.